src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -133,12 +162,14 @@ private async Task<JObject> GetRequestContentAsync(HttpRequestMessage request)
             try
             {
                 HttpRequestMessage clonedRequest = await request.CloneAsync();
-                byte[] content = await clonedRequest.Content.ReadAsByteArrayAsync();
-                return JsonConvert.DeserializeObject<JObject>(Encoding.UTF8.GetString(content, 0, content.Length));
+                
+                Stream streamContent = await clonedRequest.Content.ReadAsStreamAsync();
```

Comment:
stream is idisposable

Reply:
The question is, if you didn't create the stream, should you be the one to dispose it?